ops/20231214.md

```diff
@@ -0,0 +1,43 @@
+# 2023-12-14
+
+## Participants (use of github handle is preferred)
+- @o-otte
+- @joshuai96
+- @artificial-intelligence
+- @fkr
+- @snkaupe
+- @joshmue
+- @frostygeek
+- @scoopex
+
+
+## Recurring items
+
+- Report in the community call will be done by: @fkr
+- Responsible for the minutes is: @o-otte
+
+## Agenda
+
+### Sprint Review & Planning
+
+#### Status Page
+
+- Will be done in January with own Sprint Planning
+
+#### ADR Observability
+
+- [ADR 394](https://github.com/SovereignCloudStack/standards/pull/394) needs some rework of some sections, @o-otte is on it
+
+### Open Discussions
+
+#### Placement / Handling of compute ressources
+
+- No trivial Problem, Kolla has capabilities to configure it
+- Bandwidth limiting not there in OpenStack, can lead to peaks and exceeding of quotas
```

Comment:
what does this mean, what is missing exactly? I'm not aware of problems in this area?